api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -739,4 +739,17 @@ private void updateBypass(long userId, AccessBypassRequest request) {
     );
   }
 
+  @Override
+  public ResponseEntity<Void> deleteProfile() {
+    if (!workbenchConfigProvider.get().featureFlags.unsafeAllowDeleteUser) {
+      throw new ForbiddenException("Self account deletion is disallowed in this environment.");
+    }
+    User user = userProvider.get();
+    log.log(Level.WARNING, "User email: " + user.getEmail());
```

Comment:
Was this for debugging? I think it would probably be useful to keep this log in, but I would add a bit more context, e.g. deleting user "..."

Reply:
It was originally for debugging, but I'll add context and take credit for leaving it in! :D